thoth/user_api/api_v1.py

```diff
@@ -694,6 +696,7 @@ def list_python_packages(
         os_name=os_name,
         os_version=os_version,
         python_version=python_version,
+        starts_with=starts_with,
```

Comment:
@fridex would you recommend reusing that function: `get_python_package_version_import_packages_all` or reusing its filter?

Reply:
It would still need to have some parameter to search a package as the `list_python_packages` doesn't inherently have a way to give a package name. So maybe renaming `starts_with` to `like` and keeping the param would be better?

Also, in most query languages (including the one used in Thoth), the wildcard character is `%` and not `*`.  So I would say to change the filter to `query.filter(PythonPackageVersion.package_name.like(like))`. So the use can query for `like="%faust"`, `like="%faust%"`, `like="faust%"` to get a desired result.